test/production/jest/next-image-preload/next-image-preload.test.ts

```diff
@@ -0,0 +1,52 @@
+import { createNext, FileRef } from 'e2e-utils'
+import { NextInstance } from 'test/lib/next-modes/base'
+import { renderViaHTTP } from 'next-test-utils'
+import path from 'path'
+
+const appDir = path.join(__dirname, 'app')
+
+describe('next/jest', () => {
+  let next: NextInstance
+
+  beforeAll(async () => {
+    next = await createNext({
+      files: {
+        app: new FileRef(path.join(appDir, 'app')),
```

Comment:
```suggestion
        skipStart: true,
        app: new FileRef(path.join(appDir, 'app')),
```

I think you don't want `next start`, right?